dartagnan/src/test/resources/dartagnan-skip.csv

```diff
@@ -157,4 +157,32 @@ litmus/RISCV/SB+popaqs+NEW.litmus
 litmus/RISCV/SB+popar+poarar+NEW.litmus
 litmus/RISCV/SB+popar+poarp+NEW.litmus
 litmus/RISCV/SB+popars+NEW.litmus
-litmus/RISCV/SB+porlps+NEW.litmus
\ No newline at end of file
+litmus/RISCV/SB+porlps+NEW.litmus
+litmus/PTX/Liveness/2_threads_4_instructions/62_simple.litmus
```

Comment:
Add comment "The following tests have spinloops which are not side-effect-free (due to the use of Exch) and thus we cannot verify their liveness"